docs/2.0/reference/pipelines/configurations-as-code/index.md

```diff
@@ -320,3 +320,20 @@ authentication {
 ```
 
 In this example, Pipelines authenticates with AWS using OIDC and assumes the `role-to-assume-for-plans` role within the AWS account identified by `an-aws-account-id` when executing Terragrunt plan commands.
+
+### Environment variables block
+
+[Full Reference for Environment Variables Block](/2.0/reference/pipelines/configurations-as-code/api#env-block)
+
+The environment variables(env) block is a configuration component used by [repository](#repository-blocks) blocks to specify environment variables that will be set when executing Terragrunt commands.
+
+e.g.
+
+```hcl
+repository {
+  env {
+    TG_TF_PATH         = "opentofu"
```

Comment:
This is maybe confusing given we have docs for `tf_binary` - we're signaling two different ways to do the same thing.  I think we should encourage customers to use `tf_binary` and show a different example here